access_application.go

```diff
@@ -230,7 +230,7 @@ type SaasApplication struct {
 	NameIDFormat                  string                `json:"name_id_format,omitempty"`
 	SSOEndpoint                   string                `json:"sso_endpoint,omitempty"`
 	DefaultRelayState             string                `json:"default_relay_state,omitempty"`
-	CustomAttributes              []SAMLAttributeConfig `json:"custom_attributes,omitempty"`
```

Comment:
if you're doing this, you'll likely want to make it a pointer otherwise, you'll have issues determining empty values.

Reply:
Changed 👍 Now it will distinguish between nil and empty as would be expected